src/ContentRepository/Search/IndexingSettings.cs

```diff
@@ -78,9 +78,11 @@ public class IndexingSettings : Settings
                                 {"rtf", new RtfTextExtractor()}
                             };
 
-            var isCommunity = RepositoryVersionInfo.Instance != null &&
-                RepositoryVersionInfo.Instance.OfficialSenseNetVersion != null &&
-                RepositoryVersionInfo.Instance.OfficialSenseNetVersion.Edition == "Community";
+            //UNDONE: isCommunity is always false

```

Comment:
This can be removed, we will solve the Ent/Community difference in an upper layer, or maybe with a modified setting.